deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -8,51 +8,64 @@
     time   ${[record["log"].split(/[\n\t]+/)[0]].map! {|item| JSON.parse(item)["time"]}.join("")}
   </record>
 </filter>
+#  only match fluentd logs based on fluentd container name and label them @FLUENT
```

Comment:
we should try to see if these logs are already tagged by Fluentd itself: https://docs.fluentd.org/deployment/logging#capture-fluentd-logs

Reply:
The label `@FLUENT_LOG` uses `<match fluent.**>`  which will not stop fluentd from outputting to stdout. We will then have duplicate fluentd logs. 
Ref: https://github.com/fluent/fluentd/issues/641#issuecomment-126566460

Reply:
Did it not work starting fluentd with `-o /dev/null` to output logs to file /dev/null instead of stdout? https://docs.fluentd.org/deployment/logging#output-to-log-file